### ISSUE TITLE
Manjaro Linux: exception when mounting  borg archive in vorta

### ISSUE THREAD

**Issue by wechsler42** [borgbase/vorta#280]
**Describe the bug**
See: [https://github.com/borgbackup/borg/issues/4245](https://github.com/borgbackup/borg/issues/4245)

**To Reproduce**
1. Go to Archive Tab
2. Mount an archive
3. See error: TypeError: unpack() takes no arguments (1 given)

**Desktop (please complete the following information):**
DISTRIB_ID=ManjaroLinux
DISTRIB_RELEASE=18.0.4
DISTRIB_CODENAME=Illyria
DISTRIB_DESCRIPTION="Manjaro Linux"
Desktop: KDE or Cinnamon

**Additional context**
Workaround:
use "pip3 install --user msgpack==0.5.6" --> see [https://github.com/borgbackup/borg/issues/4245#issuecomment-489548380](https://github.com/borgbackup/borg/issues/4245#issuecomment-489548380)

**Comment by ThomasWaldmann** [comment]
Note: this is a distribution/packaging issue related to msgpack. not borg, not vorta.

the only relationship to borg is that the affected distributions ignored its msgpack requirements and upgraded msgpack to an unsupported version.

**Comment by wechsler42** [comment]
Yes, from a technical point of view, I agree. A non-experienced user reception is something like this: I installed borg and vorta, I tried to mount an archive in vorta and it threw erros on me. So why is vorta not working as intended? Well, I google around and find luckily a hint to a package I did not even know that it exists. So, this story is somehow demanding for a non-experienced user ...
However, I do not want to blame any of the software packages at all.
